client/landing/stepper/declarative-flow/internals/steps-repository/components/setup-form/index.tsx

```diff
@@ -121,7 +122,11 @@ const SetupForm = ( {
 				/>
 			</FormFieldset>
 			{ children }
-			<Button className="setup-form__submit" disabled={ isLoading } type="submit">
+			<Button
+				className={ `setup-form__submit ${ isTitleEmpty && 'disabled' }` }
```

Comment:
I don't have a preference but `classNames` looks common for these, ie: https://github.com/Automattic/wp-calypso/blob/3dc11f7426a4e9359a7a6c1bacfa40204f1c5fc2/client/landing/subscriptions/components/settings/toggle-input/toggle-input.tsx#L17